pycoal/mineral.py

```diff
@@ -362,3 +365,96 @@ def convert(cls, data_dir="", db_file="", hdr_file=""):
         library = aster_database.create_envi_spectral_library(spectrum_ids, band_info)
 
         library.save(hdr_file)
+
+class SpectralToAsterConversion:
+    
+    def __init__(self):
+        """
+            This class provides a method for converting Spectral .txt files into
+            ASTER Library Version 2.0 .txt files
+            
+            Args:
+                none
+            """
+        pass
+    
+    @classmethod
+    def convert(cls, library_filename=""):
+        """
+            This class method converts a Spectral version 7 .txt file into
+            an ASTER Library Version 2.0 .spectrum.txt file
+            
+            Spectral Library Version 7 can be downloaded here https://speclab.cr.usgs.gov/spectral-lib.html
+            
+            How to use on entire USGS Spectral Library Version 7:
+            1. Move all .txt files in Spectral Library Version 7 to one directory named 'usgs_splib07_modified'
+            2. Place this directory('usgs_splib07_modified') you just made in the pycoal/examples directory
```

Comment:
Why in Gods name would we ever want to do that.
**WE ARE TRYING TO USE THIS TOOLKIT FROM THE COMMAND LINE**... come on.